src/github.com/getlantern/proxiedsites/proxiedsites.go

```diff
@@ -33,12 +33,12 @@ var (
 )
 
 type Config struct {
-	// Global list of white-listed domains
-	Cloud []string `json:"-"`
-
 	// User customizations
 	Additions []string `json:"Additions, omitempty"`
 	Deletions []string `json:"Deletions, omitempty"`
+
+	// Global list of white-listed domains
+	Cloud []string `json:"-"`
```

Comment:
Users are more likely to want to change the Additions and Deletions in the yaml, so I moved Cloud below them to have it appear further down in the YAML.